src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -68,7 +83,55 @@ public FilterPanel(UI ui, IModel model, PanelControl parentPanelControl, int pan
                 getStyleClass().remove("panel-focused");
             }
         });
-        
+        setupKeyboardShortcuts();
+    }
+
+    private void setupKeyboardShortcuts() {
+        addEventHandler(KeyEvent.KEY_PRESSED, event -> {
+            if (MAXIMIZE_WINDOW.match(event)) {
+                ui.maximizeWindow();
+            }
+            if (MINIMIZE_WINDOW.match(event)) {
```

Comment:
Could we use `else if` here instead, so we guarantee that the bindings are mutually exclusive?